.travis.yml

```diff
@@ -48,13 +46,14 @@ branches:
  only:
    - /^release-.*$/
    - /^playtest-.*$/
-   - /^pkgtest-.*$/
+   - /^devtest-.*$/
```

Comment:
We haven't used the pkgtest tags in many years, but I frequently use `devtest` for test builds on my fork.